File: Java/백준/Bronze/10178. 할로윈의 사탕/할로윈의 사탕.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int t = in.nextInt();
        for (int i=0;i<t;i++){
            int a = in.nextInt();
            int b = in.nextInt();
            System.out.printf("You get %d piece(s) and your dad gets %d piece(s).\n",a/b,a%b);
        }
    }
}

File: C#/백준/Bronze/10178. 할로윈의 사탕/할로윈의 사탕.cs
using System;

namespace q{
    class w{
        static void Main(String[] args){
            int t = int.Parse(Console.ReadLine());
            for (int i=0;i<t;i++){
                int[] aa = Array.ConvertAll(Console.ReadLine().Split(),int.Parse);
                Console.WriteLine($"You get {aa[0]/aa[1]} piece(s) and your dad gets {aa[0]%aa[1]} piece(s).");
            }
        }
    }
}